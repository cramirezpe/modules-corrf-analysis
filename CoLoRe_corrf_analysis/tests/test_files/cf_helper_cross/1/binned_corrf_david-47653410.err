100% done........90%.......70%......60%....50%......................10%.....0%In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4. In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 9,9,4.  Time taken =   0.136 sec
countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (9, 9, 4); avg_np = 6.63e+03. Boosting bin refine factor - should lead to better performance
 Time taken =   0.241 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4.  Time taken =   0.257 sec
Using AVX kernel
........20%.30%...40%...................80%........ Time taken =     3 mins   55 secs
100% done. Time taken =     4 mins    2 secs
........90%.........70%...........40%..........20%............. Time taken =   0.223 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 9,9,4.  Time taken =   0.109 sec
countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (9, 9, 4); avg_np = 6.63e+03. Boosting bin refine factor - should lead to better performance
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4.  Time taken =   0.148 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4. 0%...10%.......30%.......50%.......60%..........80%..........90%................................30%.........10%.......0% Time taken =   0.214 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4. In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 9,9,4.  Time taken =   0.149 sec
countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (9, 9, 4); avg_np = 6.65e+03. Boosting bin refine factor - should lead to better performance
.......20%.........40%...50%.....60%...70%..80%........100% done. Time taken =     2 mins
.....................30%..........10%......In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 9,9,4.  Time taken =   0.147 sec
countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (9, 9, 4); avg_np = 6.65e+03. Boosting bin refine factor - should lead to better performance
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4.  Time taken =   0.210 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4.  Time taken =   0.126 sec
0%........20%..........40%........50%........60%.......70%....80%...90%.....100% done. Time taken =     3 mins   56 secs
