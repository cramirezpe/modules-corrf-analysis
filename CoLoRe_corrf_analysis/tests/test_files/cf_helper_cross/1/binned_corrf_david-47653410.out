verson 0.12
DEBUG:__main__:main:data:	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd.fits
data2:	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s5/catalogues/1000/norsd.fits
randoms:	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd_rand.fits
out_dir:	/global/cscratch1/sd/cramirez/NBodyKit/cross_correlations/multibias/s1_s5/nside_2/rsd_norsd/0.1_200_41/0.5_0.7/1000//1
nthreads:	64
mu_max:	1.0
nmu_bins:	100
min_bin:	0.1
max_bin:	200.0
n_bins:	41
norsd:	False
zmin:	0.5
zmax:	0.7
drq_format:	True
zmin_covd:	0.01
zmax_covd:	1.5
zstep_covd:	0.005
random_downsampling:	1
pixel_mask:	[1]
nside:	2
log_level:	DEBUG
DEBUG:__main__:main:Data files:/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd.fits
DEBUG:__main__:main:Data2 files:/global/cscratch1/sd/cramirez/NBodyKit/multibias/s5/catalogues/1000/norsd.fits
DEBUG:__main__:main:Random files:/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd_rand.fits
INFO:__main__:main:Reading file:
	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd.fits
	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s5/catalogues/1000/norsd.fits
	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd_rand.fits
DEBUG:__main__:main:Length of data cat: 428899029
DEBUG:__main__:main:Length of rand cat: 428899029
INFO:__main__:main:Reading second data cat
DEBUG:__main__:main:Length of data2 cat: 428891516
DEBUG:__main__:main:Applying pixel mask
DEBUG:__main__:main:Length of data after pixel mask: 8936279
DEBUG:__main__:main:Length of rand after pixel mask: 8942536
DEBUG:__main__:main:Length of data2 after pixel mask: 8931661
DEBUG:__main__:main:Length of data after z mask: 2147147
DEBUG:__main__:main:Length of rand after z mask: 2154210
DEBUG:__main__:main:Length of data2 after z mask: 2142749
INFO:__main__:main:Starting computation for current file:
INFO:__main__:main:Computing DD...
INFO:__main__:main:Computing DR...
INFO:__main__:main:Computing RR...
INFO:__main__:main:Computing RD...
DEBUG:__main__:main:Relative ellapsed time: 840.739343881607
